.travis.yml

```diff
@@ -14,14 +14,11 @@ language: go
 env:
   global:
     - GO_METALINTER_VERSION="v2.0.11"
-    - GLUSTERFS_CSI_DOCKERFILE="pkg/glusterfs/Dockerfile"
 
 go:
   # Kubernetes minimum_go_version can be found in this file:
   # https://github.com/kubernetes/kubernetes/blob/master/hack/lib/golang.sh
   - stable
-  - "1.10.x"  # used by OCP releases
-  - "1.9.4"   # used by CentOS 7.5.1804
```

Comment:
why these versions are removed ? We could still keep it and have a check by travis to make sure we are not broken on immediate previous versions. I see we are building docker image with 1.10, but its fine to have travis builds against `1.9.4` and `1.10.x`.

Reply:
I've removed all the non-containerized builds, cutting Travis jobs down to 1. I'm just pulling golang 'stable' for use by the linters. If the release builds w/ 1.10 (in container) always & everywhere, I don't see much reason to ensure 1.9.4 compatibility. Forward compatibility is more interesting because eventually OpenShift will advance from 1.10. My current proxy for that is "stable" for linting, but....

I'm considering removing all non-containerized stuff from Travis, and moving the linting to the containerized 1st stage. This would cut Travis CI time by only fetching vendor once instead of the current twice.

WDYT?

Reply:
I think it's fine to just containerize everything. As long as the build runs anywhere, it shouldn't matter what version of Go we build with.